pkg/util/kube/utils.go

```diff
@@ -129,6 +129,11 @@ func GetVolumeDirectory(pod *corev1api.Pod, volumeName string, pvcLister corev1l
 		return volume.Name, nil
 	}
 
+	// Pod volume is actually a HostPath
+	if volume.HostPath != nil {
```

Comment:
This section would need to be above line 125 above. Where it is now, `volume.HostPath` will never be non-nil, since any non-PVC volume source would be true above and this method will have returned `volume.Name` already.

Reply:
Actually, scratch that. You probably want this if block *inside* the `if volume.VolumeSource.PersistentVolumeClaim == nil` block, right after the CSI VolumeSource check.

Reply:
Good point, I will change that, thank you.